website/docs/cli/configuration/terminal.mdx

```diff
@@ -30,6 +30,7 @@ settings:
     pager: true          # Pager setting for all terminal output
     colors: true          # Enable colored output
     unicode: true         # Use Unicode characters in output
+    tab_width: 4          # Number of spaces for YAML indentation (default: 2)
```

Comment:
Since people copy and paste, I think we should use the recommend default in the example.

```suggestion
    tab_width: 2          # Number of spaces for YAML indentation (default: 2)
```